src/main/scala/com/ibm/couchdb/api/Documents.scala

```diff
@@ -30,40 +30,40 @@ class Documents(client: Client, db: String, typeMapping: TypeMapping) {
   val types  = typeMapping.types
   val server = new Server(client)
 
-  private def getClassName[T](obj: T): String = obj.getClass.getCanonicalName
+  private def getClassName[S](obj: S): String = obj.getClass.getCanonicalName
 
-  def create[T: upickle.Writer](obj: T): Task[Res.DocOk] = {
+  def create[S: upickle.default.Writer](obj: S): Task[Res.DocOk] = {
```

Comment:
With `default` the names become quite long, what do you think about adding aliases as in https://github.com/beloglazov/couchdb-scala/blob/updateDependencyVersions/src/main/scala/com/ibm/couchdb/core/Client.scala#L34 ?